wporg-gp-translation-events.php

```diff
@@ -97,6 +97,18 @@ public function activate(): void {
 		UNIQUE KEY `event_per_translated_original_per_user` (`event_id`,`locale`,`original_id`,`user_id`)
 		) COMMENT='Tracks translation actions that happened during a translation event'";
 		dbDelta( $create_actions_table );
+
+		// Create attendees table.
+		$create_attendees_table = "
+		CREATE TABLE `{$gp_table_prefix}event_attendees` (
+			`translate_event_attendees_id` int(11) unsigned NOT NULL AUTO_INCREMENT,
```

Comment:
The primary key is technically not needed but since the actions table above has it (and it's also not technically needed), I guessed I should probably add it here too.